### DISCUSSION TITLE
Gmail / Google Mail

### DISCUSSION THREAD

**Discussion by ANewChanceAtLife** [catppuccin/catppuccin#1853] [Port Requests]
**Are you available to help in the creation of this port? 😄 (optional)**
Yes

**Additional context 📷 (optional)**
No real context needed, but think these themes would look super cute in GMail

**Comment by isabelroses** [comment]
This is now being picked up by myself. I love the idea, thanks for bringing it to my attention.

**Reply by isabelroses** [reply]
It took longer then i thought but were finally there https://github.com/catppuccin/userstyles/pull/295

**Comment by sgoudham** [comment]
Closing as it now exists: https://github.com/catppuccin/userstyles/tree/main/styles/gmail
